packages/smooth_app/lib/pages/product/common/product_list_page.dart

```diff
@@ -231,6 +231,59 @@ class _ProductListPageState extends State<ProductListPage>
         actionModeTitle: Text(appLocalizations.compare_products_appbar_title),
         actionModeSubTitle:
             Text(appLocalizations.compare_products_appbar_subtitle),
+        actionModeActions: <Widget>[
+          IconButton(
+            icon: const Icon(Icons.delete),
+            onPressed: () async {
+              if (_selectedBarcodes.isNotEmpty) {
+                debugPrint(_selectedBarcodes.toString());
+                await showDialog<bool>(
+                  context: context,
+                  builder: (BuildContext context) {
+                    return SmoothAlertDialog(
+                      body: Text(
+                          appLocalizations.confirm_clear_selected_user_list),
+                      positiveAction: SmoothActionButton(
+                        onPressed: () async {
+                          await daoProductList.bulkSet(
+                              productList, _selectedBarcodes.toList(),
+                              include: false);
+                          await daoProductList.get(productList);
+                          setState(() {});
+                          if (!mounted) {
+                            return;
+                          }
+                          Navigator.of(context).pop();
+                        },
+                        text: appLocalizations.yes,
+                      ),
+                      negativeAction: SmoothActionButton(
+                        onPressed: () {
```

Comment:
To be consistent with the app, could you do it in a single line: `onPressed: () => Navigator…`